tools/activity/fft.py

```diff
@@ -377,8 +377,8 @@ def print_graph(x):
         print "\n############ INPUT ######################"
         print_graph(input)
 
-        print "\n############ GAUSIAN OF INPUT ############"
-        # input = apply_gausian(input, 0.1)
+        print "\n############ GAUSSIAN OF INPUT ############"
```

Comment:
This does change the width of a thing, but this line was _already_ wider than the INPUT line, so I didn't think it was worth removing trailing `#`s.